pkg/test/expectations/expectations.go

```diff
@@ -755,3 +623,9 @@ func ExpectParallelized(fs ...func()) {
 	}
 	wg.Wait()
 }
+
+func ExpectForceCleanedUpAll(ctx context.Context, c client.Client) {
+	ExpectForceCleanedUp(ctx, c, &corev1.NodeList{}, &v1.NodeClaimList{}, &corev1.PersistentVolumeClaimList{}, &corev1.PersistentVolumeList{},
```

Comment:
Optional nit.

Consider defining these values in a slice at the top of the file for improved readability